rakelib/artifacts.rake

```diff
@@ -156,8 +156,8 @@ namespace "artifact" do
   task "archives_docker" => ["prepare", "generate_build_metadata"] do
     license_details = ['ELASTIC-LICENSE']
     @bundles_jdk = true
-    create_archive_pack(license_details, "x86_64", "linux", "darwin")
-    create_archive_pack(license_details, "arm64", "linux", "darwin")
+    create_archive_pack(license_details, "x86_64", "linux")
```

Comment:
suggest to remove "darwin" from archives_docker_oss

Reply:
done